content/accessibility/developer-toolkit.mdx

```diff
@@ -0,0 +1,61 @@
+---
+title: Developer toolkit
+---
+
+A great way to find answers to specific accessibility questions is to do an online search for the component/issue in question + 'a11y', 'accessible', or 'accessibility', and scan for official websites like W3C ([https://w3.org](https://w3.org)). 
+
+**Internal resources**
+- #accessibility channel in Slack
+- Fluent community's accessibility channel in Teams (This channel is dedicated to making Fluent Design System accessible, but is home to many talented designers and developers passionate about accessibility)
+## Accessibility Insights
+One of the best things a developer can do is learn by doing, and the same goes for developing accessibly. 
+
+A helpful learning and accountability tool is [Accessibility Insights](https://accessibilityinsights.io/). Any time a developer creates a new or changed user-facing experience, Accessibility Insights should be run on the site or component to catch any accessibility errors. 
+
+### FastPass
+Within FastPass mode you have two tests to run.
```

Comment:
Can we add a sentence that tells what this is before getting into the how?

Reply:
Ah OK, I didn't realise this was within Accessibility Insights. So maybe instead of my previous comment, maybe we need a sentence before we get into the 2 modes like "Accessibility Insights can be run in two modes, depending on xyz." --> replace xyz with actual words 😄 for example "how much time you have available" or something more accurate.